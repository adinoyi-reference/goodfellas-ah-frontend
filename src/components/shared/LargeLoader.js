import React from 'react';

const LargeLoader = () => (
  <div className="large-spinner-wrapper">
    <img className="large-spinner" src="https://res.cloudinary.com/drmmqcxkc/image/upload/v1541701878/large-spinner.gif" alt="" />
  </div>
);

export default LargeLoader;
