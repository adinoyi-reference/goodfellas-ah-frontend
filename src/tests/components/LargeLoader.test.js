import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LargeLoader from '../../components/shared/LargeLoader';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<LargeLoader />);
});

describe('LargeLoader UI', () => {
  describe('render features', () => {
    test('component should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
