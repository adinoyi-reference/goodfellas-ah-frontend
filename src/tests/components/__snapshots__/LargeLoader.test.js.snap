// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LargeLoader UI render features component should render as expected 1`] = `
<div
  className="large-spinner-wrapper"
>
  <img
    alt=""
    className="large-spinner"
    src="https://res.cloudinary.com/drmmqcxkc/image/upload/v1541701878/large-spinner.gif"
  />
</div>
`;
